README.md

```diff
@@ -4,9 +4,11 @@ Gradle plugin for Embulk plugins
 Versions
 ---------
 
-| Gradle Plugin version | Supported Gradle version(s) |
-| --------------------- | --------------------------- |
-| v0.6.1                | Gradle 7.6.1                |
+| Gradle Plugin version | Expected Gradle version |
+| --------------------- | ----------------------- |
+| v0.6.2                | Gradle 7.6 (7.6.3)      |
+| v0.6.1                | Gradle 7.6 (7.6.1)      |
+| v0.6.0                | Gradle 7.6 (7.6.1)      |
 
```

Comment:
I leave my thoughts but please ignore them.

What do you think about adding this notice? (I thought about it for a while. But, both versions (7.6.2 and 7.6.3) include security fixes. It is hard to add a notice for every release.) 

> Gradle 7.6.2 and 7.6.3 were patch releases including security fixes.

Reply:
Hmm, I think it would be too much. Almost all such software can security issues, and it's hard for us to catch up with them always.

On the other hand, this note is just for information. Even while v0.6.1 is buitl with Gradle 7.6.1, v0.6.1 is expected to work with Gradle 7.6.3. Even while v0.6.2 is built with Gradle 7.6.3, v0.6.2 is expected to still work with Gradle 7.6.1.

The users of Gradle (and this plugin) are responsible to catching up with the security news of Gradle. Not our responsibility to provide the information.

Reply:
I agree with you👍. it would be too much.